<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CGAL WSPD</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/CGAL.qrc">
    <normaloff>:/cgal/logos/cgal_icon</normaloff>:/cgal/logos/cgal_icon</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="transformationAnchor">
       <enum>QGraphicsView::NoAnchor</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout">
      <item>
       <widget class="QSpinBox" name="textFrom"/>
      </item>
      <item>
       <widget class="QPushButton" name="swapButton">
        <property name="text">
         <string>&lt;-&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="textTo"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout">
      <item>
       <widget class="QPushButton" name="erasePathButton">
        <property name="text">
         <string>Erase path</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="findPathButton">
        <property name="text">
         <string>Find path</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout">
      <item>
       <widget class="QSpinBox" name="pointToDisplay"/>
      </item>
      <item>
       <widget class="QPushButton" name="displayNeighborsButton">
        <property name="text">
         <string>Display neighbors</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="eraseNeighborsButton">
        <property name="text">
         <string>Erase neighbors</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout">
      <item>
       <widget class="QCheckBox" name="biggestBoxCheck">
        <property name="text">
         <string>Take biggest box</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="edgeInsideCheck">
        <property name="text">
         <string>Take edge inside</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="numberPointsTests"/>
      </item>
      <item>
       <widget class="QPushButton" name="randomTestsButton">
        <property name="text">
         <string>Random tests</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <widget class="QTextEdit" name="results">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="fileToolBar">
   <property name="windowTitle">
    <string>File Tools</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionClear"/>
   <addaction name="actionLoadPoints"/>
   <addaction name="actionSavePoints"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Visualization Tools</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionInsertPoint"/>
   <addaction name="actionShowWspd"/>
   <addaction name="actionShowBoundingBoxes"/>
   <addaction name="actionRecenter"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionClear"/>
    <addaction name="actionLoadPoints"/>
    <addaction name="actionSavePoints"/>
    <addaction name="separator"/>
    <addaction name="actionExport_SVG"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionInsertRandomPoints"/>
    <addaction name="actionSetSeparationRatio"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionInsertPoint"/>
    <addaction name="actionShowWspd"/>
    <addaction name="actionShowBoundingBoxes"/>
    <addaction name="separator"/>
    <addaction name="actionRecenter"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAboutCGAL">
   <property name="text">
    <string>About &amp;CGAL</string>
   </property>
  </action>
  <action name="actionExport_SVG">
   <property name="text">
    <string>&amp;Export SVG...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionInsertRandomPoints">
   <property name="text">
    <string>&amp;Insert random points</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionSetSeparationRatio">
   <property name="text">
    <string>&amp;Set separation ratio</string>
   </property>
  </action>
  <action name="actionInsertPoint">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons/Input.qrc">
     <normaloff>:/cgal/Input/inputPoint.png</normaloff>:/cgal/Input/inputPoint.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Insert point</string>
   </property>
   <property name="toolTip">
    <string>Insert point</string>
   </property>
   <property name="statusTip">
    <string>Left: Insert vtx</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="icons/File.qrc">
     <normaloff>:/cgal/fileToolbar/fileNew.png</normaloff>:/cgal/fileToolbar/fileNew.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Clear</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionShowBoundingBoxes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons/Triangulation_2.qrc">
     <normaloff>:/cgal/Triangulation_2/Voronoi_diagram_2.png</normaloff>:/cgal/Triangulation_2/Voronoi_diagram_2.png</iconset>
   </property>
   <property name="text">
    <string>Show &amp;Bounding Boxes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionShowWspd">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons/Triangulation_2.qrc">
     <normaloff>:/cgal/Triangulation_2/Delaunay_triangulation_2.png</normaloff>:/cgal/Triangulation_2/Delaunay_triangulation_2.png</iconset>
   </property>
   <property name="text">
    <string>Show &amp;WSPD</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionLoadPoints">
   <property name="icon">
    <iconset resource="icons/File.qrc">
     <normaloff>:/cgal/fileToolbar/fileOpen.png</normaloff>:/cgal/fileToolbar/fileOpen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Load Points...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionSavePoints">
   <property name="icon">
    <iconset resource="icons/File.qrc">
     <normaloff>:/cgal/fileToolbar/fileSave.png</normaloff>:/cgal/fileToolbar/fileSave.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save Points...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionRecenter">
   <property name="icon">
    <iconset resource="icons/Input.qrc">
     <normaloff>:/cgal/Input/zoom-best-fit</normaloff>:/cgal/Input/zoom-best-fit</iconset>
   </property>
   <property name="text">
    <string>Re&amp;center the viewport</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons/File.qrc"/>
  <include location="resources/CGAL.qrc"/>
  <include location="icons/Triangulation_2.qrc"/>
  <include location="icons/Input.qrc"/>
 </resources>
 <connections/>
</ui>
